Makefile

```diff
@@ -26,15 +26,17 @@ update-readme:
 
 update: update-readme
 
-test-go:
+test-build-dir:
+	mkdir -p $(PWD)/build/test-results
```

Comment:
That's funny as @stuartnelson3 found it in the same time [here](https://github.com/elastic/elastic-package/pull/565#discussion_r740779179) :)

I would ask you or @stuartnelson3 to extract changes in Makefile to a separate PR.

Reply:
Ok, @stuartnelson3 did this already (thanks!). Would you mind adjusting your PR?

Reply:
Done.